### ISSUE TITLE
Test Group Icons have Insufficient Context 

### ISSUE THREAD

**Issue by mgifford** [ckan/ckan#7185]
**CKAN version**

https://demo.ckan.org/group/test-group

## Describe the bugs

**Purple**
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[data-format="geojson"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Test Group - Groups - CKAN Demo - https://demo.ckan.org/group/test-group

Element path: a[data-format="geojson"]

Snippet: <a href="/dataset/sample-dataset-1" class="label label-default" data-format="geojson">GeoJSON</a>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.45 (foreground color: #ffffff, background color: #9855e0, font size: 7.9pt (10.5px), font weight: bold). Expected contrast ratio of 4.5:1

**Green 1**
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[data-format="kml"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Test Group - Groups - CKAN Demo - https://demo.ckan.org/group/test-group

Element path: a[data-format="kml"]

Snippet: <a href="/dataset/sample-dataset-1" class="label label-default" data-format="kml">KML</a>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 2.64 (foreground color: #ffffff, background color: #7aae3d, font size: 7.9pt (10.5px), font weight: bold). Expected contrast ratio of 4.5:1

**Green 2**
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[data-format="wms"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Test Group - Groups - CKAN Demo - https://demo.ckan.org/group/test-group

Element path: a[data-format="wms"]

Snippet: <a href="/dataset/sample-dataset-1" class="label label-default" data-format="wms">WMS</a>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 1.91 (foreground color: #ffffff, background color: #adc717, font size: 7.9pt (10.5px), font weight: bold). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 107.0.1418.35

====

This accessibility issue was found using Accessibility Insights for Web 2.35.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by amercader** [comment]
If I'm not mistaken this report refers to the format pills, specifically the geospatial ones (geojson, wms and kml) which are defined in [ckanext-geoview](https://github.com/ckan/ckanext-geoview/blob/master/ckanext/geoview/public/css/geo-resource-styles.css)
![Screenshot 2022-11-29 at 15-43-01 Test Group - Groups - CKAN Demo](https://user-images.githubusercontent.com/200230/204559455-4626b90d-90d6-4f68-8145-d1c8bdcf4c6d.png)

**Comment by mgifford** [comment]
Yes. Correct. 
![Screenshot of buttons](https://user-images.githubusercontent.com/116832/208750272-74bd58ec-1e86-4397-856b-8f88e94d6e42.png)
